go.mod

```diff
@@ -4,7 +4,7 @@ go 1.19
 
 require (
 	github.com/stretchr/testify v1.7.2
-	github.com/waku-org/go-zerokit-rln-apple v0.0.0-20240116135015-f6f595c7b8ef
+	github.com/waku-org/go-zerokit-rln-apple v0.0.0-20240117094748-68b4162e8fd7
```

Comment:
temporally pointing to https://github.com/waku-org/go-zerokit-rln-apple/pull/3
until merged.

TODO: update when merged.